### PR TITLE
On delete close row without animation

### PR DESCRIPTION
**Pull request by zavin27**
fixes issue #96 

just call closeRowWithoutAnimation() on the row like in the SwipeListExample/example.js row 38.

### CODE REVIEW

**Comment by jemise111** [comment]
Hey @zavin27 thanks for this! Before I review would you mind turning off your linter and resubmitting with just the lines of code you modified?

File: components/SwipeRow.js [hunk 1]
```diff
@@ -80,14 +80,10 @@ class SwipeRow extends Component {
 	}
 
 	shouldComponentUpdate(nextProps, nextState) {
-		if (this.state.hiddenHeight !== nextState.hiddenHeight ||
+		return !!(this.state.hiddenHeight !== nextState.hiddenHeight ||
```
[the remark this answers is not in the document]
Comment (zavin27) [reply components/SwipeRow.js]: done, I've reverted it to the way it was although it was the same with fewer lines of code :)

[the remark this answers is not in the document]
Comment (jemise111) [reply components/SwipeRow.js]: Yeah but it's for readability's sake + I like to keep blame information the same whenever possible. Thanks!

**Comment by thanhluantl2304** [comment]
Dear mr @jemise111 , I still face this issue. I don't use SwipeRow component so need Swipe ROw or not? Please explain that for me! THank you!

**Pull request merged**